src/components/Table.jsx

```diff
@@ -0,0 +1,606 @@
+        return (
+            <>
+                <span className="mr-2">
+                    Assignee
+                </span>
+                <span>
+                <button onClick={handleAssigneeSort}>
+                    <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+
+    const handleUpdatedBySortHeader=()=>{
+        return (
+            <>
+                <span className="mr-2">
+                    Updated By
+                </span>
+                <span>
+                <button onClick={handleUpdatedBySort}>
+                    <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+
+    const handleUpdatedAtSortHeader=()=>{
+        return (
+            <>
+                <span className="mr-2">
+                    Updated At
+                </span>
+                <span>
+                <button onClick={handleUpdatedAtSort}>
+                   <img alt="ArrowSort icon" src={ArrowSort}/>                    
+                </button>
+                </span>
+            </>
+        );
+    }
+const handleFilterIcon=()=>{
+    return(
+        <span>
+            <button onClick={() => setShowFilters(!showFilters)}>
+                 {!showFilters ? <div className="bg-black text-white text-sm rounded-full border h-8 w-8 flex justify-center items-center"> 
+                   <i className="pi pi-filter" style={{fontSize: '0.8rem'}}/> 
+                 </div> :
+                 <div className="bg-white text-black text-sm rounded-full border h-8 w-8 flex justify-center items-center"> 
+                   <i className="pi pi-filter" style={{fontSize: '0.8rem'}}/> 
+                 </div>}
+            </button>                
+        </span> 
+    )
+}
+
+    const handleAssigneeSort=()=>{
+        setCurrentSort("assigneeSort")
+        setAssigneeSort(assigneeSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleUpdatedAtSort=()=>{
+        setCurrentSort("updatedAtSort")
+        setUpdatedAtSort(updatedAtSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleUpdatedBySort=()=>{
+        setCurrentSort("updatedBySort")
+        setUpdatedBySort(updatedBySort == "desc" ? "asc" : "desc")
+    }
+
+    const handleStatusSort=()=>{
+        setCurrentSort("statusSort")
+        setStatusSort(statusSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleBrandSort=()=>{
+        setCurrentSort("brandSort")
+        setBrandSort(brandSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleTitleSort=()=>{
+        setCurrentSort("titleSort")
+        setTitleSort(titleSort == "desc" ? "asc" : "desc")
+    }
+
+    const handleStyleSort=()=>{
+        setCurrentSort("styleSort")
+        setStyleSort(styleSort == "desc" ? "asc" : "desc")
+    }
+
+
+    const handleEdit=(rowData)=>{
```

Comment:
This and the next two methods look similar. 
Merge.

Reply:
addressed